tests/db/model/generator/methods.js

```diff
@@ -85,9 +84,9 @@ describe('tests/db/model/generator/methods.js >', () => {
         isActive: true,
         possibleCollectors: [coll2.name, coll3.name],
       })
-      .then(() => {
-        expect(generator1.currentCollector.name).to.equal(collector2.name);
-        expect(generator1.currentCollector.id).to.equal(collector2.id);
+      .then((updated) => {
+        expect(updated.currentCollector.name).to.equal(collector2.name);
+        expect(updated.currentCollector.id).to.equal(collector2.id);
```

Comment:
Is this all just for readability?

Reply:
The call to updateWithCollectors returns the updated and reloaded object, so we want to be verifying the values on that reloaded object. That was the only thing I found here that would have been responsible for the flapper, so I changed it on the flapping test and other obvious places we were doing the same thing.

Reply:
Oh ok. Hmm... we already have a reference to the object, so I wouldn't think it would make a difference. Maybe a sequelize bug? In any case it's fine either way so it makes sense to try this.